src/internal/m365/onedrive/handlers.go

```diff
@@ -86,7 +90,9 @@ type RestoreHandler interface {
 	GetItemsByCollisionKeyser
 	GetRootFolderer
 	ItemInfoAugmenter
+	NewDrivePagerer
```

Comment:
Can we add short descriptions for these interfaces?

Reply:
I can add descriptions to the interface declarations, yes.